src/main/kotlin/com/cjbooms/fabrikt/generators/controller/MicronautControllerInterfaceGenerator.kt

```diff
@@ -164,6 +211,7 @@ class MicronautControllerInterfaceGenerator(
                 .builder(MicronautImports.PATH_VARIABLE)
         }.let {
             it.addMember("value = %S", parameter.oasName)
+            // here it will be added to each parameter
```

Comment:
**question:** Is this an open to-do?

Reply:
Nope. Good eye! This is a leftover from when I was trying to understand the code